src/models/hashmodel.h

```diff
@@ -104,6 +104,23 @@ class HashModel : public QAbstractTableModel
         return index(row, column);
     }
 
+    static QString formatSymbolBinaryString(const QString& symbol, const QString& binary)
```

Comment:
also, these helper functions shouldn't be in class scope, but rather in a helper header and then used wherever needed